backend/migrating/v2/README.md

```diff
@@ -1,17 +1,113 @@
-# V2 Migrations and Commands
+# V2 Migration Guide
 
 This folder contains scripts and management commands for performing data migrations from v1 to v2 for the Unstract Multitenancy system.
+From version `v0.90.3` onwards, there have been signifcant changes in how data is stored. This warrants a data migration and the below steps can come in handy.
```

Comment:
The version mentioned here (v0.90.3) is likely to change, as it will reflect the release of v2 without the feature flag. From that point onward, all v2 changes will be fully applied.
cc: @hari-kuriakose

Reply:
Yes @muhammad-ali-e we need to change it when we introduce that PR - I can leave a placeholder maybe instead?

Reply:
ok @chandrasekharan-zipstack , We can change it later then.